File: capture.js
const { chromium } = require('playwright');
const path = require('path');

(async () => {
  const browser = await chromium.launch({ headless: true });
  const page = await browser.newPage({
    viewport: { width: 1440, height: 900 }
  });
  
  const htmlPath = path.resolve(__dirname, 'dist/index.html');
  await page.goto('file://' + htmlPath, { waitUntil: 'networkidle' });
  await page.waitForTimeout(2000);
  
  await page.screenshot({ 
    path: path.resolve(__dirname, 'anne-cox-website.png'),
    fullPage: true 
  });
  
  console.log('Screenshot saved to anne-cox-website.png');
  await browser.close();
})();

File: capture.py
from playwright.sync_api import sync_playwright
import os

with sync_playwright() as p:
    browser = p.chromium.launch(headless=True)
    page = browser.new_page(viewport={'width': 1440, 'height': 900})
    
    # Open the built HTML file directly
    html_path = '/Users/alexanderkehaya/.openclaw/workspace/anne-cox-site/dist/index.html'
    page.goto(f'file://{html_path}')
    page.wait_for_timeout(2000)
    
    page.screenshot(
        path='/Users/alexanderkehaya/.openclaw/workspace/anne-cox-site/anne-cox-website.png',
        full_page=True
    )
    print('Screenshot saved!')
    browser.close()
